data.md

```diff
@@ -139,6 +139,18 @@ WebAssembly values are either integers or floating-point numbers, of 32- or 64-b
  // -----------------------------
 ```
 
+**TODO**: This needs to actually be parsed and added to the `Int` and `Float` sorts, and be added to the prelude.
```

Comment:
The way this TODO is phrased is a bit confusing. Isn't parsing and adding to the `Int` and `Float` sorts exactly what these syntax definitions are doing?